src/scicookie/{{cookiecutter.project_slug}}/coc/NUMFOCUS_adapted_coc.md

```diff
@@ -0,0 +1,231 @@
+# NumFOCUS-Adapted Code of Conduct
```

Comment:
Here I have doubts because the other files that are as coc options, the name appears as I placed it. That is why I did it this way. But I am going to change it